api/apps/api/test/project-scenarios.e2e-spec.ts

```diff
@@ -136,5 +136,28 @@ describe('ScenariosModule (e2e)', () => {
 
       expect(resources).toBeUndefined();
     });
+
+    it('should not allow to create scenario with invalid marxan properties', async () => {
```

Comment:
Albeit less immediately useful, I would also ensure that if we supply valid `input.dat` metadata we can successfully create or update a scenario (may be useful to catch some gross regressions).